[supervisor] close of closed channel panic

### Bug description

While terminating workspace, noticed this in log output from workspace pod:
```
Connection closed by UNKNOWN port 65535                            
debug1: channel 0: free: server-session, nchannels 1               
Close session: user gitpod from UNKNOWN port 65535 id 0            
debug1: do_cleanup                                                 
Transferred: sent 691708, received 1028712 bytes                   
Closing connection to UNKNOWN port 65535                           
mm_request_send: write: Broken pipe                                
debug1: do_cleanup                                                 
panic: close of closed channel                                     
                                                                   
goroutine 81 [running]:                                            
github.com/gitpod-io/gitpod/supervisor/cmd.glob..func3.2()         
    github.com/gitpod-io/gitpod/supervisor/cmd/init.go:87 +0xb8    
created by github.com/gitpod-io/gitpod/supervisor/cmd.glob..func3  
    github.com/gitpod-io/gitpod/supervisor/cmd/init.go:83 +0x79e   
```
Seems to be this line:
https://github.com/gitpod-io/gitpod/blob/4c6b2246a1892129a938d38d6da21f6a8a24d9c2/components/supervisor/cmd/init.go#L87

### Steps to reproduce

close workspace

### Workspace affected

_No response_

### Expected behavior

_No response_

### Example repository

_No response_

### Anything else?

_No response_

--- a/components/supervisor/cmd/init.go
+++ b/components/supervisor/cmd/init.go
@@ -84,7 +84,6 @@ var initCmd = &cobra.Command{
 				defer close(terminationDone)
 				slog.TerminateSync(ctx, runCommand.Process.Pid)
 				terminateAllProcesses(ctx, slog)
-				close(supervisorDone)
 			}()
 			// wait for either successful termination or the timeout
 			select {

